scripts/output/single/ar6Climate.R

```diff
@@ -0,0 +1,152 @@
+library(madrat)
+library(remind2)
+library(quitte)
+library(piamInterfaces)
+library(yaml)
+library(lucode2)
+library(readxl) # GA: Wont be necessary after https://github.com/iiasa/climate-assessment/pull/43 goes into release
+
+############################# BASIC CONFIGURATION #############################
+gdx_name     <- "fulldata.gdx"             # name of the gdx
+cfg_name     <- "cfg.txt"                  # cfg file for getting file paths
+
+
+if(!exists("source_include")) {
+   # Define arguments that can be read from command line
+   outputdir <- "."
+   readArgs("outputdir", "gdx_name", "gdx_ref_name", "gdx_refpolicycost_name")
+}
+
+gdx           <- file.path(outputdir, gdx_name)
+cfg_path      <- file.path(outputdir, cfg_name)
+logfile       <- file.path(outputdir, "climate.log") # specific log for python steps
+scenario <- getScenNames(outputdir)
+remind_reporting_file <- file.path(outputdir,paste0("REMIND_generic_", scenario,".mif"))
+
+print(getwd())
+############################# PREPARING EMISSIONS INPUT #############################
+
+# Read the cfg to get the location of MAGICC-related files
+cfg <- read_yaml(cfg_path)
+
+# Read the GDX and run reportEmi
+# gdxpath <- "fulldata.gdx"
+cat(date()," ar6Climate.R: Running reportEmi \n")
+emimag <- reportEmi(gdx)
+
+# Convert to quitte and add metadata
+emimif <- as.quitte(emimag)
+emimif["scenario"] <- scenario #TODO: Get scenario name from cfg
+
+# Write the raw emissions mif
+# TODO: This wouldn't be necessary if we added an option to generateIIASASubmission
```

Comment:
generateIIASASubmission should work with a quitte object:
https://github.com/pik-piam/piamInterfaces/blob/master/R/generateIIASASubmission.R#L5-L6
```
> #' @param mifs path to mif files or directories with mif files of a REMIND run,
> #'             or quitte object
```
and if you set `outputFilename` to `NULL`, the function also returns a quitte object, so you can continue working with that directly.

So I think
```
ar6mif <- remind_reporting_file %>%
  filter(region %in% c("GLO", "World")) %>%
  generateIIASASubmission(mapping = "AR6", outputFilename = NULL, logFile = NULL, iiasatempate = "magicc_variables.yaml") %>%
  mutate(region = factor("World"))
```
should basically do what you need. Why do you prefer to pass a csv, not an xlsx? Then, you could just use `write.IAMCxlsx` from the `quitte` package to write it.